cypress/commons/constants/generic/TestConstants.js

```diff
@@ -45,3 +45,8 @@ export const API_REGEX = {
 };
 
 export const SCENARIO_RUN_IN_PROGRESS = 'Scenario run in progress...';
+
+export const WEBAPP_URL_REGEX = {
+  SCENARIO_PAGE: new RegExp(`/${PAGE_NAME.SCENARIO}`),
+  SCENARIO_PAGE_WITH_ID: new RegExp(`${PAGE_NAME.SCENARIO}/.*`),
```

Comment:
Do we want the SCENARIO_PAGE regex to only match URLs without scenario ids ? If yes, then the regex should end with `$` to make sure it does not match URLs with scenario ids.
Also, we could use a "group capture" in the SCENARIO_PAGE_WITH_ID regex instead of .* (e.g. `${PAGE_NAME.SCENARIO}/(s-[\\w]+)` )

Reply:
Great, thank you ! :slightly_smiling_face: